### ISSUE TITLE
Rotation based on uses, not time

### ISSUE THREAD

**Issue by CoolDouble** [zhaow-de/rotating-tor-http-proxy#10]
Is there a way to rotate the IP manually after N requests?

**Comment by abdelrahman-mh** [comment]
No!, HAProxy here make a roundrobin algorithm for your HTTP requests on a single port 3128 (inside the container), and he not provide algorithm to give each proxy N requests and move to next one. and i think the roundrobin is do the same thing but he distribute your requests to all the Proxies.

else you may change all setup
